account_banking_payment_transfer/model/payment_line.py

```diff
@@ -160,28 +159,9 @@ def is_zero(total):
                 cr, uid, transit_move_line.company_id.currency_id, total)
 
         line_ids = [transit_move_line.id, torec_move_line.id]
-        if torec_move_line.reconcile_partial_id:
-            line_ids = [
-                x.id for x in
-                torec_move_line.reconcile_partial_id.line_partial_ids
-                ] + [transit_move_line.id]
-
-        total = move_line_obj.get_balance(cr, uid, line_ids)
-        vals = {
-            'type': 'auto',
-            'line_id': is_zero(total) and [(6, 0, line_ids)] or [(6, 0, [])],
-            'line_partial_ids': (is_zero(total) and
-                                 [(6, 0, [])] or
-                                 [(6, 0, line_ids)]),
-            }
-
-        if torec_move_line.reconcile_partial_id:
-            reconcile_obj.write(
-                cr, uid, [torec_move_line.reconcile_partial_id.id],
-                vals, context=context)
-        else:
-            reconcile_obj.create(
-                cr, uid, vals, context=context)
+        move_line_obj.reconcile_partial(cr, uid, line_ids, type='manual',
+                                        context=context)
+
         for line_id in line_ids:
             workflow.trg_trigger(
                 uid, 'account.move.line', line_id, cr)
```

Comment:
I think this trigger can be removed as it is called by reconcile().
https://github.com/odoo/odoo/blob/8.0/addons/account/account_move_line.py#L1074